docs/api/native-theme.md

```diff
@@ -63,6 +63,15 @@ Your application should then always use `shouldUseDarkColors` to determine what
 A `boolean` for if the OS / Chromium currently has high-contrast mode enabled
 or is being instructed to show a high-contrast UI.
 
+### `nativeTheme.shouldUseDarkColorsForSystemIntegratedUI` _macOS_ _Windows_ _Readonly_
+
+A `boolean` property indicating whether or not the system theme has been set to dark or light.
+
+On Windows this property distinguishes between system and app light/dark theme, returning
+`true` if the system theme is set to dark theme and `false` otherwise. On macOS the return
+value will be the same as `nativeTheme.shouldUseDarkColors`, as macOS does not distinguish
```

Comment:
It's technically possible for Mac apps to opt out of this and have a different color scheme from the OS. Specifically they can opt out of Dark Mode and always run in Light Mode. https://developer.apple.com/documentation/appkit/choosing-a-specific-appearance-for-your-macos-app

Reply:
What's tripping me up is this part of these docs:

> as macOS does not distinguish between system and app theme.

It's possible for a macOS app to opt for light mode even if the system is in dark mode. In that case I believe `nativeTheme.shouldUseDarkColors` will reflect the app's theme rather than the system's. And it seems like a Mac app would also want to be able to query the underlying system theme for things like the app icon to use in the Tray or Dock.

Reply:
Oh I see. You're saying "macOS does not distinguish between system and app theme" is just wrong then, as evidenced by the docs you linked to. @codebytere heads up.

Reply:
I removed it - the implementation returns `shouldUseDarkColors` on macOS so that's true regardless.